### PR TITLE
Added Popup Sample

### PR DESCRIPTION
**Pull request by MKellyEsri**
Hyperlinks which pass latatitude and longitude values dynamically

### CODE REVIEW

File: popup/url-withlocation.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+if (EndGoal == "Google Directions") {
+	BaseUrl = "https://www.google.com/maps/dir/?api=1&origin=Current+Location&destination="
+} else if (EndGoal == "Google Panoramic") {
+	BaseUrl = "https://www.google.com/maps/@?api=1&map_action=pano&viewpoint="
+} else if (EndGoal == "Google Pin") {
+	BaseUrl = "https://maps.google.com/maps?t=k&q=loc:"
+} else {
+	Console("Invalid End Goal")
+}
+
```
Comment (ekenes) [inline popup/url-withlocation.md]: Perhaps we could use [Decode()](https://developers.arcgis.com/arcade/function-reference/logical_functions/#decode) here to condense the code a bit?

So this:

```js
var BaseUrl;
if (EndGoal == "Google Directions") {
	BaseUrl = "https://www.google.com/maps/dir/?api=1&origin=Current+Location&destination="
} else if (EndGoal == "Google Panoramic") {
	BaseUrl = "https://www.google.com/maps/@?api=1&map_action=pano&viewpoint="
} else if (EndGoal == "Google Pin") {
	BaseUrl = "https://maps.google.com/maps?t=k&q=loc:"
} else {
	Console("Invalid End Goal")
}
```

Becomes...

```js
var BaseUrl = Decode( EndGoal,
  "Google Directions", "https://www.google.com/maps/dir/?api=1&origin=Current+Location&destination=",
  "Google Panoramic", "https://www.google.com/maps/@?api=1&map_action=pano&viewpoint=",
  "Google Pin", "https://www.google.com/maps/@?api=1&map_action=pano&viewpoint=",
  "Invalid"
);

if (BaseUrl == "Invalid"){
  Console( EndGoal + " is an invalid value for EndGoal.");
}
```

It looks a little more readable that way, plus the end user gets a more descriptive message when an invalid EndGoal is provided. 

WDYT? Test it out and see if it works OK.

**Comment by MKellyEsri** [comment]
Thanks for taking a look. Great suggestion - I’ll revise on Monday.

**Comment by MKellyEsri** [comment]
Just made suggested revisions @ekenes :wink: 2369b38dace2ac31956f4a70cb9364572fd00d5d

**Comment by ekenes** [comment]
Thanks! Made a few minor formatting updates. Merging...

**Comment by MKellyEsri** [comment]
Great stuff - thanks @ekenes.

**Comment by ekenes** [comment]
@MKellyEsri I just tested your webmap and it looks like the link for the Google Pin and the Google Panormaic are the same: 

Panoramic: https://www.google.com/maps/@?api=1&map_action=pano&viewpoint=
Pin:              https://www.google.com/maps/@?api=1&map_action=pano&viewpoint=

```js
var BaseUrl = Decode( EndGoal,
  "Google Directions", "https://www.google.com/maps/dir/?api=1&origin=Current+Location&destination=",
  "Google Panoramic", "https://www.google.com/maps/@?api=1&map_action=pano&viewpoint=",
  "Google Pin", "https://www.google.com/maps/@?api=1&map_action=pano&viewpoint=",
  "Invalid"
);
```

**Comment by MKellyEsri** [comment]
Apologies - the Google Pin Url somehow got changed. It's the same in the master: https://github.com/Esri/arcade-expressions/blob/master/popup/url-withlocation.md. Should I open another pull request or do you want to make the change?

The Google Pin BaseUrl needs to be changed to "https://maps.google.com/maps?t=k&q=loc:" in the Decode function.

I have revised the Web Map.

**Comment by ekenes** [comment]
I'll go ahead and make the change. Thanks!

**Comment by ekenes** [comment]
Done.

**Pull request merged**